tools/travis/src/installdeps.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Script to install all the LaTeXML dependencies
+set -e
+
+# Use perlbrew
+source /usr/local/perlbrew/etc/bashrc
+
+# Update apt
+apt-get update
+
+# Install system dependencies
+apt-get install -y \
+    libdb-dev \
+    libxml2-dev \
+    libxslt1-dev \
+    zlib1g-dev
```

Comment:
One tricky bit is to remember to keep this file up-to-date as we keep developing if dependencies are added. I guess the good news is that the travis tests will fail when it is out of date, so we can't quite forget to :+1: